tests/components/wallbox/test_config_flow.py

```diff
@@ -115,17 +97,10 @@ async def test_form_validate_input(hass: HomeAssistant) -> None:
         DOMAIN, context={"source": config_entries.SOURCE_USER}
     )
 
-    with requests_mock.Mocker() as mock_request:
-        mock_request.get(
-            "https://user-api.wall-box.com/users/signin",
-            json=authorisation_response,
-            status_code=HTTPStatus.OK,
-        )
-        mock_request.get(
-            "https://api.wall-box.com/chargers/status/12345",
-            json=test_response,
-            status_code=HTTPStatus.OK,
-        )
+    with patch("wallbox.Wallbox.authenticate", return_value=None,), patch(
```

Comment:
```suggestion
    with patch("wallbox.Wallbox.authenticate", return_value=None), patch(
```

Stray comma ?

Reply:
stray, forgot to remove after removing a side_effect clause